AUTHORS.rst

```diff
@@ -1,25 +1,64 @@
 People
 ------
    
-Core Contributors
-=================
+Organisation Admins
+===================
 
-* Ankur Ankan
+* Ankur Ankan <ankurankan@gmail.com>
 
-* Abinash Panda
+* Abinash Panda <mailme.abinashpanda@gmail.com>
 
-Other Contributors
-==================
+Contributors
+============
 
-* Shashank Garg
+(in order of the date of their first contribution made)
 
-* Shikhar Nigam
+* Ankur Ankan <ankurankan@gmail.com>
 
-* Pranjal Mittal
+* Shikhar Nigam <snigam3112@gmail.com>
 
-GSoC 2014 Students
-==================
+* Abinash Panda <mailme.abinashpanda@gmail.com>
 
-* Navin Chandak
+* William Lyon <wlyon@quantpost.com>
 
-* Jaspreet
+* Navin Chandak <navinchandak92@gmail.com>
+
+* Jaspreet Singh <jaspreetsingh112@gmail.com>
+
+* nehasoni <neha.soni.ece10@itbhu.ac.in>
+
+* Jaidev Deshpande <deshpande.jaidev@gmail.com>
+
+* Abhijeet Kislay <abhijeetkislay@gmail.com>
+
+* palashahuja <abhor902@gmail.com>
+
+* The Gitter Badger <badger@gitter.im>
+
+* Saket Choudhary <saketkc@gmail.com>
+
+* all3fox <all3fox@gmail.com>
+
+* Vivek Jain <vivek425ster@gmail.com>
```

Comment:
This is the real name, remove the other vivek425ster, see both names are associated with same email